### DISCUSSION TITLE
Is there any way the last opened character sheets and libraries can be restored? If not - it's a feature request!

### DISCUSSION THREAD

**Discussion by Marv93** [richardwilkes/gcs#662] [Ideas]
I just started using GCS and it was great to be able to quickly rebuild my existing character sheets in GCS. Then I closed GCS and opened it again - only to see an empty Document Workspace. I was surprised but then thought "ok there has to be a setting to restore the stuff I had open when I closed GCS" but I didn't find such a setting. If there is one, my question is: where is this setting? If not here is my feature request:

Similar to browsers GCS should have a setting that specifies which documents should be opened when starting GCS:
- a predefined set of documents (can be empty, is the current default and may be the default when this setting is introduced)
- the documents that were open when GCS was closed the last time

A similar feature would be to save and load whole Document Workspaces.

**Comment by richardwilkes** [comment]
As you noted, GCS doesn't currently do this for you. I'm not sure if there is already an issue here for it (I've not looked) -- if not, creating one would make it more likely I eventually add such a feature, as I look through the issues when I'm in need of something to work on and don't have anything specific in mind.

**Comment by Marv93** [comment]
This issue https://github.com/richardwilkes/gcs/issues/361 seems to be the best fit for this. I will therefore close this discussion.
